packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -497,6 +513,59 @@ export class JobService {
     },
   };
 
+  private createCvatActions: Record<JobRequestType, CvatAction> = {
+    [JobRequestType.HCAPTCHA]: {
+      getElementsCount: async () => 0,
+    },
+    [JobRequestType.FORTUNE]: {
+      getElementsCount: async () => 0,
+    },
```

Comment:
if the actions is called createCvatActions, we should not include FORTUNE and HCAPTCHA. Otherwise we might call it getElementsActions or getElementsCountAction

Reply:
What do you think about to organize it likewise: 
`private createDatasetActions: Record<JobRequestType, DatasetAction>`
`- getElementsCount`